src/DynamoCoreWpf/Properties/Resources.en-US.resx

```diff
@@ -2949,6 +2949,12 @@ This package will be unloaded after the next Dynamo restart.</value>
   <data name="InfoBubbleInfo" xml:space="preserve">
     <value>Info</value>
   </data>
+  <data name="AutodeskSamplePackage" xml:space="preserve">
+    <value>Autodesk sample</value>
+  </data>
```

Comment:
I believe this resource string has been removed in latest `master`, it might appear as a conflict when merging master but you kept it. Would you remove it?